silx/test/utils.py

```diff
@@ -243,6 +239,57 @@ def temp_dir():
         shutil.rmtree(tmp_dir)
 
 
+# Simulate missing library context
+class EnsureImportError(object):
```

Comment:
`ensure_import_error` should be better here as it is not in `silx.gui`.

Reply:
Even for a class? Is there a specific good practice for naming context manager classes ?

Reply:
My bad, I read too fast.

I could not found a guideline for that... and looking at Python doc does not help, you find both CamelCase and `_` (especially with contextlib which allows to define a context manager with a function, but even with classes...):
- https://docs.python.org/3/library/contextlib.html#using-a-context-manager-as-a-function-decorator
- https://www.python.org/dev/peps/pep-0343/